pkg/utils/grub.go

```diff
@@ -160,6 +160,34 @@ func (g Grub) Install(target, rootDir, bootDir, grubConf, tty string, efi bool,
 			}
 		}
 
+		var foundFonts bool
```

Comment:
The whole thing can be extracted to an `installGrubFonts` function (or some other fitting name).

Reply:
just used once in here, not reusable. I dont think it needs the poor visibility of extracting into a function :)

Reply:
Extracting code to a function has other benefits (testing separately, making sure it's self-contained and that no variables are touched by this code when they shouldn't, etc). Self-contained tasks should live in self-contained functions but we don't seem to follow this rule anywhere else in the code. This is not the proper place to introduce new rules. I'm fine leaving it as is.